ci/check_style.sh

```diff
@@ -9,7 +9,7 @@ rapids-logger "Create checks conda environment"
 rapids-dependency-file-generator \
   --output conda \
   --file_key checks \
-  --matrix "" | tee env.yaml
+  --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch)" | tee env.yaml
```

Comment:
I know the dependencies list isn't affected by the Python version, but it'd be nice to align with other repos here since it won't negatively impact the environment generation.
```suggestion
  --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch);py=${RAPIDS_PY_VERSION}" | tee env.yaml
```

Reply:
Works for me